crates/katana/runner/src/deployer.rs

```diff
@@ -1,38 +1,84 @@
-use anyhow::{Context, Ok, Result};
+use std::env;
+
+use anyhow::{anyhow, Context, Ok, Result};
+use clap::Parser;
+use dojo_lang::compiler::DojoCompiler;
+use dojo_lang::plugin::CairoPluginRepository;
+use dojo_lang::scarb_internal::compile_workspace;
+use scarb::compiler::CompilerRepository;
+use scarb::core::{Config, TargetKind};
+use scarb::ops::CompileOpts;
+use sozo::args::{Commands, SozoArgs};
+use sozo::ops::migration;
 use starknet::core::types::FieldElement;
+use starknet::macros::felt;
 use tokio::process::Command;
 
 use crate::KatanaRunner;
 
+async fn prepare_migration_args(args: SozoArgs) -> Result<()> {
+    // Preparing config, as in https://github.com/neotheprogramist/dojo/blob/fec863cb0375a684edd9e7e76c8fdeb9ebf386e6/bin/sozo/src/cli_entry.rs#L29
```

Comment:
Can we update these urls to join to the dojoengine org rather than a fork?

Reply:
sure, updated